runtime_test.go

```diff
@@ -2998,7 +2998,7 @@ func TestExceptionStringifyError(t *testing.T) {
 			exceptionVal: func() Value {
 				return newStringValue("hello")
 			},
-			expectedString: "\"hello\"",
+			expectedString: "hello",
```

Comment:
This is so much better!

[q] I'm not familiar with the usage of StringifyError but is it possible that this could break existing customer functions because they are expecting the old patterns?

Reply:
Nope. We introduced this function for error handling in eventbridge so it isn’t used anywhere yet. Looking forward I figured it would be generally useful for situations where an exception could contain an object and we care about that data in baas :)

Reply:
Oh perfect then! Send it!